charts/fybrik/values.yaml

```diff
@@ -465,6 +508,9 @@ opaConnector:
     name: opa-connector
 
   tls:
+    # MinVersion contains the minimum TLS version that is acceptable.
+    # If not provided, minimal supported TLS protocol is taken as the minimum.
```

Comment:
we have to list all possible values in the comments, plus my previous comment about self-described values

Reply:
done, Thanks